### PR TITLE
Add CDC ingestion as a Cloud Run Service

### PR DESCRIPTION
**Pull request by mayaspivak**
I created a cloud run service under the run_ingestion directory. This copies the common directory from data_ingestion for now, but I can try to share the code when running the docker build in a github action. You will notice that the main.py is very similar to the GCF on with some extra request handling that GCF does automatically and the added cdc ingestion.

### CODE REVIEW

File: run_ingestion/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+# Use the official lightweight Python image.
+# https://hub.docker.com/_/python
+FROM python:3.8-slim
```
Comment (jenniebrown) [inline run_ingestion/Dockerfile:4]: Python 3.8 is beta in cloud functions, but I didn't see equivalent documentation for Cloud Run. Do you know if it's fully supported?

File: run_ingestion/main.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+from flask import Flask, request
+app = Flask(__name__)
+
+
+# Data source name literals. These correspond to a specific data ingestion workflow.
+_HOUSEHOLD_INCOME = 'HOUSEHOLD_INCOME'
+_CDC_COVID_DEATHS = 'CDC_COVID_DEATHS'
+
+@app.route('/', methods=['POST'])
+def ingest_data():
```
Comment (jenniebrown) [inline run_ingestion/main.py:15]: Could you add a comment about the return type?

File: run_ingestion/main.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    if 'url' not in event_dict or 'gcs_bucket' not in event_dict or 'filename' not in event_dict:
+        logging.error("Pubsub data must contain fields 'url', 'gcs_bucket', and 'filename'")
+        return ('', 400)
+    
+    data_id = event_dict['id']
+    logging.info(f'Ingesting {data_id} data')
+    if data_id == _HOUSEHOLD_INCOME:
+        upload_household_income(event_dict['url'], event_dict['gcs_bucket'], event_dict['filename'])
+    else:
+        url_file_to_gcs(event_dict['url'], '', event_dict['gcs_bucket'], event_dict['filename'])
```
Comment (jenniebrown) [inline run_ingestion/main.py]: I think the second argument to url_file_to_gcs should technically be an empty dict instead of empty string

**Review by jenniebrown — CHANGES_REQUESTED** [review]
This is great,  just a couple minor comments

File: run_ingestion/main.py (hunk 2 above)
Comment (mayaspivak) [reply run_ingestion/main.py:15]: Done

File: run_ingestion/main.py (hunk 3 above)
Comment (mayaspivak) [reply run_ingestion/main.py]: Done.

File: run_ingestion/Dockerfile (hunk 1 above)
Comment (mayaspivak) [reply run_ingestion/Dockerfile:4]: Cloud Run doesn't have the same runtime restrictions since users do the build themselves.

**Review by jenniebrown — APPROVED** [review]

**Pull request merged**